### ISSUE TITLE
Select-all button on the network caps admin panel.

### ISSUE THREAD

**Issue by zenhack** [sandstorm-io/sandstorm#2564]
I've got about 30 leaked network caps on my dev system right now, and a good number on prod too thanks to zenhack/irc-idler#8.  It would be nice to be able to clean these up more quickly -- maybe a select all button on the network caps admin panel?

//cc @paulproteus

**Comment by dwrensha** [comment]
Note that the capabilities should automatically get deleted when you delete the grain the owns them. (You'll need to permanently delete the grain, not just move it to the trash.)

**Comment by zenhack** [comment]
Yeah, part of the issue is with prod I've been restarting IRC Idler periodically when it gets into a weird state (e.g. zenhack/irc-idler#13), so the grains never actually get deleted at all; I've been using the same ones for several weeks at least. Per the bug I linked above I end up leaking the cap on every restart. This *is* something of a weird scenario, since you wouldn't expect to have a large number of network caps per grain if things were working correctly.

**Comment by kentonv** [comment]
It sounds like the real problem here is that save/restore isn't working for you? Are you currently doing a new powerbox request every time the grain or Sandstorm restarts? Seems like the priority should be on fixing that?

**Comment by zenhack** [comment]
Yeah, that's the bigger problem, this was more of a "oh, this would be
nice" thing I noticed when working around it. But yes, currently needing
to allocate a new ipNetwork cap every time the app starts, and haven't
found time to sit down and debug it.
